### ISSUE TITLE
Update copyright headers of all files

### ISSUE THREAD

**Issue by erciccione** [haveno-dex/haveno#35]
All files have a copyright header which says `This file is part of Bisq.[...]`.

All these headers will have to say `This file is part of Haveno.[...]`. I have a PR read, but on a second thought it's probably better to wait until we start to heavily trim the code of Bisq content we don't need. A lot of file will be deleted, no reason to change their copyright header now.

**Comment by newsyp** [comment]
Got it, I'll sit on standby!

**Comment by premek** [comment]
this script could be used to do this when its the right time

```
grep -rl "* This file is part of Bisq." | xargs -d '\n'  sed -i 's/* This file is part of Bisq./* This file is part of Haveno./g'
grep -rl "* Bisq is free software: you can redistribute it and/or modify it" | xargs -d '\n'  sed -i 's/* Bisq is free software: you can redistribute it/* Haveno is free software: you can redistribute it/g'
grep -rl "* Bisq is distributed in the hope that" | xargs -d '\n' sed -i 's/* Bisq is distributed in the hope that/* Haveno is distributed in the hope that/g'
grep -rl "* along with Bisq. If not" | xargs -d '\n' sed -i 's/* along with Bisq. If not/* along with Haveno. If not/g'
```

**Comment by newsyp** [comment]
How exactly can I use that script?

Is that part of the bash interface?

**Comment by erciccione** [comment]
@newsyp you just have to run the commands listed in a bash shell. Better wait until #40 is merged, since a lot of files will be removed and doesn't make sense to change their headers now.

**Comment by erciccione** [comment]
This is resolved by #193
